swagger/api.yaml

```diff
@@ -36,7 +36,16 @@ paths:
   "/catalog/v1/catalogs/{id}/courses":
       $ref: "https://raw.githubusercontent.com/edx/course-discovery/649a4cb/api.yaml#/endpoints/v1/catalogCourses"
 
-# edX extension point. Lists the vendors in use and their specific 
+  # Enterprise IDA
+  # TODO: update `saleem-latif/ENT-238` to release tag once `https://github.com/edx/edx-enterprise/pull/70` is merged
```

Comment:
I'm going to wait to merge this PR until these refs can be pointed at a clearer location. Note that we've been using exact git hashes, not tags, to guarantee that what's deployed **exactly matches** what's referenced in GitHub at all times. Once you've merged the edx-enterprise PR, please use the git hash of the release. (We only update the api-manager deployment when there are changes to this repository, so we don't want to get into a state where you've changed your tag and the Swagger document no longer aligns with what's deployed.)